core/src/mill/define/Task.scala

```diff
@@ -44,28 +44,30 @@ abstract class Task[+T] extends Task.Ops[T] with Applyable[Task, T]{
   def self = this
 }
 
-trait NamedTask[+T] extends Task[T]{
+trait NamedTask[+T] extends Task[T] {
   def ctx: mill.define.Ctx
-  def label = ctx.segment match{case Segment.Label(v) => v}
+  def label = ctx.segment match { case Segment.Label(v) => v }
   override def toString = ctx.segments.render
 }
-trait Target[+T] extends NamedTask[T]{
+trait Target[+T] extends NamedTask[T] {
   override def asTarget = Some(this)
   def readWrite: RW[_]
 }
 
-object Target extends TargetGenerated with Applicative.Applyer[Task, Task, Result, mill.util.Ctx] {
+object Target
+    extends TargetGenerated
+    with Applicative.Applyer[Task, Task, Result, mill.util.Ctx] {
 
-  implicit def apply[T](t: T)
-                       (implicit rw: RW[T],
-                        ctx: mill.define.Ctx): Target[T] = macro targetImpl[T]
+  implicit def apply[T](t: T)(implicit rw: RW[T],
+                              ctx: mill.define.Ctx): Target[T] =
```

Comment:
I think this is inferior to the original. Having everything laid out vertically in one nice column is better than having the first row be horizontally and then subsequent args laid out vertically